.github/workflows/security-review.yml

```
@@ -0,0 +1,230 @@
+          fi
+
+      - name: Next.js security headers check
+        run: |
+          # Next.js設定のセキュリティヘッダーをチェック
+          if [ -f "next.config.ts" ] || [ -f "next.config.js" ]; then
+            echo "Checking Next.js security configuration..."
+            node -e "
+              const fs = require('fs');
+              const configFile = fs.existsSync('next.config.ts') ? 'next.config.ts' : 'next.config.js';
+              const content = fs.readFileSync(configFile, 'utf8');
+              
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)